sim/dex-moves.ts

```diff
@@ -25,25 +25,36 @@ export type MoveTarget =
 
 /** Possible move flags. */
 interface MoveFlags {
+	allyanim?: 1; // The move plays its animation when used on an ally.
+	assist?: 1; // Cannot be selected by Assist.
 	bypasssub?: 1; // Ignores a target's substitute.
 	bite?: 1; // Power is multiplied by 1.5 when used by a Pokemon with the Ability Strong Jaw.
 	bullet?: 1; // Has no effect on Pokemon with the Ability Bulletproof.
 	charge?: 1; // The user is unable to make a move between turns.
 	contact?: 1; // Makes contact.
+	copycat?: 1; // Cannot be selected by Copycat.
 	dance?: 1; // When used by a Pokemon, other Pokemon with the Ability Dancer can attempt to execute the same move.
 	defrost?: 1; // Thaws the user if executed successfully while the user is frozen.
 	distance?: 1; // Can target a Pokemon positioned anywhere in a Triple Battle.
+	encore?: 1; // Encore fails if target used this move.
+	futuremove?: 1; // Targets a slot, and in 2 turns damages that slot.
 	gravity?: 1; // Prevented from being executed or selected during Gravity's effect.
 	heal?: 1; // Prevented from being executed or selected during Heal Block's effect.
+	instruct?: 1; // Cannot be repeated by Instruct.
+	mefirst?: 1; // Cannot be selected by Me First.
+	mimic?: 1; // Cannot be copied by Mimic.
 	mirror?: 1; // Can be copied by Mirror Move.
-	allyanim?: 1; // The move has an animation when used on an ally.
+	mustpressure?: 1; // Additional PP is deducted due to Pressure when it ordinarily would not.
 	nonsky?: 1; // Prevented from being executed or selected in a Sky Battle.
+	parentalbond?: 1; // Cannot be made to hit twice via Parental Bond.
```

Comment:
considering the purpose of this flag is to prevent it from triggering on parental bond, wouldn't something like `unbound` make more sense? similar to sleep talk etc flags

Reply:
I don't like 'unbound' but 'non[thing]' or 'no[thing]' or 'cant[thing]' are all options I think are reasonable and am very open to any name suggestions.